Backend/Remora.Discord/Remora.Discord.Core/Snowflake.cs

```diff
@@ -139,6 +139,24 @@ public int CompareTo(Snowflake other)
             return this.Value.CompareTo(other.Value);
         }
 
+        /// <summary>
+        /// Compares two snowflakes, for equality.
+        /// </summary>
+        /// <param name="left">The left operand.</param>
+        /// <param name="right">The right operand.</param>
+        /// <returns>true if the operands are equal, false otherwise.</returns>
+        public static bool operator ==(Snowflake left, Snowflake right)
+            => left.Equals(right);
+
+        /// <summary>
+        /// Compares two snowflakes, for inequality.
+        /// </summary>
+        /// <param name="left">The left operand.</param>
+        /// <param name="right">The right operand.</param>
+        /// <returns>false if the operands are equal, true otherwise.</returns>
+        public static bool operator !=(Snowflake left, Snowflake right)
+            => left.Equals(right);
```

Comment:
I believe you're missing a negation operator here - this does the same thing as `==`

Reply:
Herp.